docs/source/scenarios/citi_bike.rst

```diff
@@ -753,3 +753,65 @@ random mode, we hope this could help you learn how to use the environment interf
 
 Jump to `this notebook <https://github.com/microsoft/maro/tree/master/notebooks/bike_repositioning/interact_with_environment.ipynb>`_
 for a quick experience.
+
+~~~~~~
+
+Inter-view
+~~~~~~~~~~
+
+For Inter-view, user could view top-k stations with different
+attributes.
+
+.. figure:: ..\images\visualization\dashboard\citi_bike_inter_epoch.gif
+   :alt: citi\_bike\_inter\_epoch
+
+Intra-view
+~~~~~~~~~~
+
+Intra-view is divided into two dimensions according to time and space.
+
+If user choose to view information by station, it means that attributes
+of all snapshots within a selected station would be displayed. By
+changing the option "station index", user could view data of different
+stations. By changing the option "Snapshot Sampling Ratio", Users can
+freely adjust the sampling rate. For example, if there are 100 snapshots
```

Comment:
Can user just select the first 30 snapshots for visualization?

Reply:
no. data sampling is uniform